src/styles/starter-page-templates-editor.scss

```diff
@@ -0,0 +1,86 @@
+@mixin screen-reader-text() {
+	border: 0;
+	clip: rect(1px, 1px, 1px, 1px);
+	clip-path: inset(50%);
+	height: 1px;
+	margin: -1px;
+	overflow: hidden;
+	padding: 0;
+	position: absolute;
+	width: 1px;
+	word-wrap: normal !important;
+}
+.st-template-selector__modal {
```

Comment:
@getdave Have you tried just leaving this at the standard modal size?

Reply:
I missed this btw @mtias. Only just seen